README.md

```diff
@@ -127,12 +127,11 @@ Come help us make PostHog even better. We're growing like crazy, [and would love
 
```

Comment:
One thing I'm not too sure about is the "We're growing like **crazy**" bit, since that's maybe _slightly_ too enthusiastic for what it is. 😄 Just a nitpick though.

Reply:
Hmm yeah that's been a phrase that's been used all over our website, and perhaps it made more sense when we had more open roles. I do think it is true, but maybe in the context of hiring it currently is a bit much